src/app/public/modules/inline-delete/inline-delete.component.spec.ts

```diff
@@ -22,7 +24,6 @@ import {
 } from './inline-delete-type';
 
 describe('Inline delete component', () => {
-  const ANIMATION_TIMEOUT: number = 401;
```

Comment:
Removed this timeout because it's precarious. Instead, I imported the `NoopAnimationsModule`.

Reply:
I think the animation has to operate due to the functions being called via the animation callout. This won't fire if the NoopAnimationModule is used

Reply:
The `NoopAnimationModule` calls through all of the animation callbacks, but without the transition time. To verify, you can pull it into app-extras to see it working but without the transitions. It's intended to be used within unit tests so that you don't have to wait for transitions to happen but still test the functionality.